pkgs/shells/bash/ble.sh/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib, stdenv, git, fetchgit }:
+
+stdenv.mkDerivation {
+  name = "ble.sh";
+  version = "0.3.3";
+
+  nativeBuildInputs = [ git ];
```

Comment:
Why is git needed for a... line editor?

Reply:
Also, it should be after `src`.

Reply:
It's a build dependency.

Reply:
On latest unstable ble.sh need it to ensure submodule is fetched. v0.3.3 does not require it thou.

Reply:
After removing git, it is unable to be built anymore although `fetchSubmodules = true`. I'll leave this.

Reply:
> git submodule update

If ble.sh works inside the store or the sandbox this will not work. If we fetch the submodules anyway we don't need git and should rather remove that line of code.

Reply:
Git is a dependency of `make` here. It's not a runtime-dependency.

Reply:
@SuperSandro2000 Let me correct myself. I have written above that

> I guess we don't need it as far as we specify `fetchSubmodules = true`?

but I was forgetting that the build-time dependency on `git` is not only for fetching submodules. We also need `git` to determine the embedded version string of `ble.sh`. The command `git` is called by [this line](https://github.com/akinomyoga/ble.sh/blob/a45077599d5c48d946de6e9b3c4baaaae9fc2633/ble.pp#L64) and [this line](https://github.com/akinomyoga/ble.sh/blob/a45077599d5c48d946de6e9b3c4baaaae9fc2633/ble.pp#L1490) at the build time.